gamestonk_terminal/cryptocurrency/defi/graph_view.py

```diff
@@ -0,0 +1,263 @@
+    sortby: str
+        Key by which to sort data
+    descend: bool
+        Flag to sort data descending
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df = graph_model.get_uni_tokens(skip=skip)
+    df_data = df.copy()
+
+    df = df.sort_values(by=sortby, ascending=descend)
+
+    df[["totalLiquidity", "tradeVolumeUSD"]] = df[
+        ["totalLiquidity", "tradeVolumeUSD"]
+    ].applymap(lambda x: very_long_number_formatter(x))
+
+    if gtff.USE_TABULATE_DF:
+        print(
+            tabulate(
+                df.head(limit),
+                headers=df.columns,
+                floatfmt=".2f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            ),
+            "\n",
+        )
+    else:
+        print(df.to_string, "\n")
+
+    export_data(
+        export,
+        os.path.dirname(os.path.abspath(__file__)),
+        "tokens",
+        df_data,
+    )
+
+
+def display_uni_stats(export: str = "") -> None:
+    """Displays base statistics about Uniswap DEX. [Source: https://thegraph.com/en/]
+    [Source: https://thegraph.com/en/]
+
+    Parameters
+    ----------
+
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df = graph_model.get_uniswap_stats()
+    df_data = df.copy()
+
+    if gtff.USE_TABULATE_DF:
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".2f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            ),
+            "\n",
+        )
+    else:
+        print(df.to_string, "\n")
+
+    export_data(
+        export,
+        os.path.dirname(os.path.abspath(__file__)),
+        "stats",
+        df_data,
+    )
+
+
+def display_recently_added(
+    top: int = 20,
+    days: int = 7,
+    min_volume: int = 20,
+    min_liquidity: int = 0,
+    min_tx: int = 100,
+    sortby: str = "created",
+    descend: bool = False,
+    export: str = "",
+) -> None:
+    """Displays Lastly added pairs on Uniswap DEX.
+    [Source: https://thegraph.com/en/]
+
+    Parameters
+    ----------
+    top: int
+        Number of records to display
+    days: int
+        Number of days the pair has been active,
+    min_volume: int
+        Minimum trading volume,
+    min_liquidity: int
+        Minimum liquidity
+    min_tx: int
+        Minimum number of transactions
+    sortby: str
+        Key by which to sort data
+    descend: bool
+        Flag to sort data descending
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df = graph_model.get_uniswap_pool_recently_added(
+        last_days=days,
+        min_volume=min_volume,
+        min_liquidity=min_liquidity,
+        min_tx=min_tx,
+    )
+    df_data = df.copy()
+
+    df = df.sort_values(by=sortby, ascending=descend)
+
+    df[["volumeUSD", "totalSupply"]] = df[["volumeUSD", "totalSupply"]].applymap(
```

Comment:
This operation should be done in the model I think, can you take a look @DidierRLopes?

Reply:
I'm ok having it here to be fair so that the raw values get untouched in `df_data` and the user can see what's the raw content we got before doing some "pretty" processing.

Reply:
Yes exactly, it's because if I add it in model, then you can't sort values as numbers because it will be converted to strings like 1M, 15k etc.